vtex-apps/store-video
# Adding playsInline prop to video players

#### What problem is this solving?

This PR adds the ability to set the video property `playsInline`. This prop is specially useful for iOS devices to autoplay the content properly. It defaults to true if `autoPlay` is enabled.

I've taken the "inline" translations from [here](https://developers.google.com/youtube/player_parameters).

#### How to test it?

Go to the site editor on the workspace below and on the Story 1 - Toggle media, there's a video. Since it already has autoPlay enabled, it has the `displaysinline` property on the dom, no matter the value on the `playsInline` toggle. Try disabling autoplay and enabling the playsInline toggle and inspect the element to see that it still includes the `playsinline` property.

[Workspace](https://icarovtexvideo--muji.myvtex.com/admin/cms/site-editor/how-to-muji-face-soap?__siteEditor&__bindingAddress=www.mujionline.eu/uk)

#### Screenshots or example usage:

<img width="261" alt="Screen Shot 2021-03-05 at 16 56 39" src="https://user-images.githubusercontent.com/8127610/110166915-c3b62e00-7dd3-11eb-99c6-0a8c0198ef57.png">

#### Related to / Depends on

[store-media#4](https://github.com/vtex-apps/store-media/pull/4)

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/3ornjPby6F2oHVkJQA/giphy.gif)
